[Bug]: Storybook for Vue tutorial Error

### Describe the bug

I just follow the Storybook for Vue tutorial to run the following commands:

# Clone the template
```sh
npx degit chromaui/intro-storybook-vue-template taskbox
cd taskbox
```

# Install dependencies
```sh
yarn
```

# Start the component explorer on port 6006:
```sh
yarn storybook
```

I get the error:
```
info => Serving static files from ././public at /
info => Starting manager..
✘ [ERROR] Could not resolve "@storybook/addon-toolbars/manager"

    node_modules/@storybook/addon-essentials/dist/toolbars/manager.mjs:1:11:
      1 │ export*from"@storybook/addon-toolbars/manager";
        ╵            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  The Yarn Plug'n'Play manifest forbids importing "@storybook/addon-toolbars"
  here because it's not listed as a dependency of this package:

    ../../../.pnp.cjs:38:33:
      38 │           "packageDependencies": [\
         ╵                                  ~~

  You can mark the path "@storybook/addon-toolbars/manager" as external to
  exclude it from the bundle, which will remove this error.
✘ [ERROR] Could not resolve "@storybook/addon-outline/manager"
✘ [ERROR] Could not resolve "@storybook/addon-backgrounds/manager"
✘ [ERROR] Could not resolve "@storybook/addon-actions/manager"
✘ [ERROR] Could not resolve "@storybook/addon-viewport/manager"
✘ [ERROR] Could not resolve "@storybook/addon-measure/manager"
✘ [ERROR] Could not resolve "@storybook/addon-controls/manager"
✘ [ERROR] Could not resolve "@storybook/global"
✘ [ERROR] Could not resolve "@storybook/instrumenter"
✘ [ERROR] Could not resolve "polished"
```

After I delete .yarnrc.yml file & .yarn folder in the taskbox folder
.yarnrc.yml file:
```
nodeLinker: node-modules
```
i run yarn && yarn storybook again. I get this error
```
✘ [ERROR] Could not resolve "@vue/compiler-core"

    .yarn/__virtual__/@storybook-vue3-virtual-31e430be77/0/cache/@storybook-vue3-npm-7.2.3-e07567b7c0-b9d3213fb8.zip/node_modules/@storybook/vue3/dist/config.mjs:4:26:
      4 │ import { baseParse } from '@vue/compiler-core';
        ╵                           ~~~~~~~~~~~~~~~~~~~~

  The Yarn Plug'n'Play manifest says this package has a peer dependency on "@vue/compiler-core",
  but the package "@vue/compiler-core" has not been installed:

    .pnp.cjs:5095:35:
      5095 │             ["@vue/compiler-core", null],\
           ╵                                    ~~~~

  You can mark the path "@vue/compiler-core" as external to exclude it from the bundle, which
  will remove this error.
```
After i run yarn add @vue/compiler-core --dev && yarn storybook, it works


### To Reproduce

_No response_

### System

```shell
System:
    OS: macOS 13.2.1
    CPU: (8) arm64 Apple M1 Pro
  Binaries:
    Node: 18.17.1 - /usr/local/bin/node
    Yarn: 3.6.1 - /opt/homebrew/bin/yarn
    npm: 9.6.7 - /usr/local/bin/npm
  Browsers:
    Chrome: 115.0.5790.170
    Safari: 16.3
```


### Additional context
Maybe something wrong when using node-modules for the dependancies? Anyone can help on this?

--- a/code/renderers/vue3/package.json
+++ b/code/renderers/vue3/package.json
@@ -51,6 +51,7 @@
     "@storybook/global": "^5.0.0",
     "@storybook/preview-api": "workspace:*",
     "@storybook/types": "workspace:*",
+    "@vue/compiler-core": "^3.0.0",
     "lodash": "^4.17.21",
     "ts-dedent": "^2.0.0",
     "type-fest": "~2.19",
@@ -60,13 +61,11 @@
     "@digitak/esrun": "^3.2.2",
     "@types/prettier": "2.7.2",
     "@vitejs/plugin-vue": "^4.4.0",
-    "@vue/compiler-core": "^3.3.4",
     "typescript": "^5.3.2",
     "vue": "^3.2.47",
     "vue-tsc": "latest"
   },
   "peerDependencies": {
-    "@vue/compiler-core": "^3.0.0",
     "vue": "^3.0.0"
   },
   "engines": {

